src/main/kotlin/no/skatteetaten/aurora/boober/service/openshift/OpenShiftClient.kt

```diff
@@ -198,14 +196,17 @@ class OpenShiftClient(
      * @param labelSelectors examples: name=someapp, name!=someapp (name label not like), name (name label must be set)
      */
     fun getByLabelSelectors(kind: String, namespace: String, labelSelectors: List<String>): List<JsonNode> {
-        val queryString = urlEncode(Pair("labelSelector", labelSelectors.joinToString(",")))
+        val queryString = labelSelectors.joinToString(",")
```

Comment:
Vi måtte urlencode her tidligere siden vi kalte en exchange metode som ikke gjorde det for oss. Nå er det fikset så da kan vi ikke gjøre det her.